pkg/config/shared.go

```diff
@@ -44,7 +44,8 @@ type Base struct {
 
 // Log represents configuration for the logger.
 type Log struct {
-	Level log.Level `name:"level" description:"The minimum level log messages must have to be shown"`
+	Format string    `name:"format" description:"Log format to write (old, console, json)"`
```

Comment:
Unless you already have a diff that removes `old` here and in usage, it would be helpful to include a TODO reference to remove it in 3.14